shopinvader/services/settings.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.base_rest import restapi
+from odoo.addons.component.core import Component
+
+
+class ExportSettingsService(Component):
+    """Shopinvader service to expose allowed settings"""
+
+    _inherit = [
+        "base.shopinvader.service",
+    ]
+    _name = "shopinvader.settings.service"
+    _usage = "settings"
+    _description = __doc__
+
+    def get(self):
+        return {
+            **self._get_countries(),
+            **self._get_titles(),
+            **self._get_industries(),
+            **self._get_currencies(),
+            **self._get_languages(),
+        }
+
+    def _jsonify_fields_country(self):
+        return [
+            "name",
+            "code",
+            "id",
+        ]
+
+    def _get_countries(self):
+        countries = self.shopinvader_backend.allowed_country_ids.jsonify(
+            self._jsonify_fields_country()
+        )
+        return {"countries": countries}
+
+    @restapi.method(
+        [(["/countries"], "GET")],
+        auth="api_key",
+    )
+    def countries(self):
+        return self._get_countries()
+
+    def _jsonify_fields_title(self):
+        return [
+            "id",
+            "name",
+        ]
+
+    def _get_titles(self):
+        titles = self.shopinvader_backend.partner_title_ids.jsonify(
+            self._jsonify_fields_title()
+        )
+        return {"titles": titles}
+
+    @restapi.method(
+        [(["/titles"], "GET")],
+    )
```

Comment:
All these methods should be public.
```suggestion
    @restapi.method(
        [(["/titles"], "GET")],
        auth="public"
    )
```